### PR TITLE
tile38 1.12.0

### PR DESCRIPTION
**Pull request by yocontra**
basically a copy of the redis plist

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by contra** [comment]
ping @ilovezfs who did all of the other tile38 updates, not sure if you care about this or not but seems right to tag you

**Comment by contra** [comment]
Keep it open.

File: Formula/tile38.rb [hunk 1]
```diff
@@ -32,11 +32,39 @@ def post_install
...
   def caveats; <<~EOS
-    Data directory created at #{datadir}. To start the server:
-        tile38-server -d #{datadir}
+    To connect: tile38-cli
+    EOS
+  end
 
-    To connect:
-        tile38-cli
+  plist_options :manual => "tile38-server -d #{HOMEBREW_PREFIX}/var/tile38/data"
```
Comment (ilovezfs) [inline Formula/tile38.rb]: does `#{var}` is equivalent to `#{HOMEBREW_PREFIX}/var`

Comment (ilovezfs) [reply Formula/tile38.rb]: hm I guess that doesn't work in `plist_options`. ok

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Review by ilovezfs — APPROVED** [review]

**Comment by commitay** [comment]
Thanks @contra!

**Pull request closed without merging**